### ISSUE TITLE
Make highlighted types copyable

### ISSUE THREAD

**Issue by yoavbls** [yoavbls/pretty-ts-errors#29]

**Comment by costamatheus97** [comment]
We would need to replace the codicon with some other solution, any ideas?

**Comment by djmisterjon** [comment]
a btn to copytoclipboard ?
```ts
navigator.clipboard.writeText(text);
      console.log('Content copied to clipboard');
```

```ts
navigator.permissions.query({ name: "write-on-clipboard" }).then((result) => {
  if (result.state == "granted" || result.state == "prompt") {
    alert("Write access granted!");
  }
});
```

**Comment by yoavbls** [comment]
@costamatheus97 VSCode HTML sanitization is very strict. I don't see any other options that they'll make it less strict. I opened an issue and PR for that:
https://github.com/microsoft/vscode/issues/180496

**Comment by djmisterjon** [comment]
> @costamatheus97 VSCode HTML sanitization is very strict. I don't see any other options that they'll make it less strict. I opened an issue and PR for that: [microsoft/vscode#180496](https://github.com/microsoft/vscode/issues/180496)
> 
> @djmisterjon If they'll reject the PR it's a good option. Does it work with the VSCode command link?

idk sry, it work on my nwjs app, never tested with electron (used by vscode)

**Comment by yoavbls** [comment]
Until things will be changed you can use the hack here:
https://github.com/yoavbls/pretty-ts-errors/blob/main/docs/hide-original-errors.md

**Comment by MichaelDimmitt** [comment]
Feature requests: #29 and #155 
Seems to be related to #156 and #152 

Possibly resolved by them?

**Comment by kevinramharak** [comment]
@MichaelDimmitt Yes, definitely good to point out here that [`0.7.0`](https://github.com/yoavbls/pretty-ts-errors/releases/tag/v0.7.0) now has a new tab + panel version of the error message, which does not have the limitations that the hover versions have.

I still want to keep this one open and make a new attempt to get the VS Code team to fix this. It's a minor change, and even with all the votes, we never got a response.
Unfortunately, their issue backlog is swamped, and a lot of good and highly voted feature requests are still open, so I do not have big hopes. But it's worth another try.
